expression/expression.go

```diff
@@ -488,3 +488,28 @@ func SplitCNFItems(onExpr Expression) []Expression {
 func SplitDNFItems(onExpr Expression) []Expression {
 	return splitNormalFormItems(onExpr, ast.OrOr)
 }
+
+// EvaluateExprWithNull sets inner table columns in a expression as null and calculate the final result of the scalar function.
```

Comment:
`sets columns in the schema as null`
This function doesn't know if `schema` is inner table.